source/includes/code-examples/FindManyAsync.cs

```diff
@@ -0,0 +1,96 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+using MongoDB.Driver.Linq;
+
+namespace CSharpExamples.UsageExamples;
+
+public class FindManyAsync
+{
+    private static IMongoCollection<Restaurant> _restaurantsCollection;
+    private static string _mongoConnectionString = "<Your MongoDB URI>";
+
+    public static void Main(string[] args)
+    {
+        Setup();
+
+        // Find multiple documents using builders
+        Console.WriteLine("Finding documents with builders...:");
```

Comment:
Since one of the recent versions of C#, you can add `using static System.Console;` to the top of the file, and then you don't have to repeat `Console` every time. We shouldn't do this if it makes the examples confusing, though. What do you think?

Reply:
I didn't know that was an option. I like it, I don't think it makes things more confusing, I'm good with adding it